protocol/messenger_communities.go

```diff
@@ -520,6 +520,22 @@ func (m *Messenger) RequestToJoinCommunity(request *requests.RequestToJoinCommun
 		}
 	}()
 
+	// Activity center notification
+	notification := &ActivityCenterNotification{
+		ID:               types.FromHex(requestToJoin.ID.String()),
+		Type:             ActivityCenterNotificationTypeCommunityRequest,
+		Timestamp:        requestToJoin.Clock,
```

Comment:
@cammellos this `requestToJoin.Clock` returns timestamp `1666182043`, same as uint64(time.Now().Unix()). But whisper timestamp is usually like 1666181801000. What i should use to obtain timestamp like that?

Reply:
Well, I've found several ways to generate timestamp in current codebase:
1. utils.GetUnixEpoch()
2. s.m.getTimesource().GetCurrentTime(),
3. uint64(time.Now().Unix())
4. time.Unix(time.Now().Unix(), 0),
5. time.Now().UnixNano()

Option 2 `getTimesource().GetCurrentTime()` works good for me